ERCS/eip-send-tx-conditional.md

```diff
@@ -0,0 +1,158 @@
+---
+eip: send-tx-cond
+title: Conditional send transaction RPC
+description: JSON-RPC API for block builders allowing users to express preconditions for transaction inclusion
+author: Dror Tirosh (@drortirosh), Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Shahaf Nacson (@shahafn)
+discussions-to:
```

Comment:
Please create a discussions topic in Eth Magicians: https://ethereum-magicians.org/c/ercs/57